### DISCUSSION TITLE
Hyperparameter Tuning

### DISCUSSION THREAD

**Discussion by junsu0302** [junsu0302/ARIA#13] [DNN]
# 하이퍼파라미터 조정

하이퍼파라미터 조정은 다음과 같은 주요 하이퍼파라미터 값을 조정한다.
- Learning Rate : 가중치를 업데이트하는 크기 결정
- Batch Size : 한 번의 업데이트에 사용되는 데이터 샘풀 수 결정
- Number of Epochs : 전체 데이터셋을 학습하는 반복 횟수 결정
- Regularization Coefficient : L1, L2 정규화 등에서 패널티 항의 크기 결정
- Dropout Rate : 드롭아웃에서 뉴런을 비활성화할 확률

대표적인 하이퍼파라미터 조정 방법은 다음과 같다.
- 그리드 탐색
- 랜덤 탐색
- 베이지안 최적화
- 기울기 기반 최적화
- 랜덤 그리드 탐색
